scrapy/core/scheduler.py

```diff
@@ -119,7 +120,7 @@ def _dqpush(self, request):
             return
         try:
             self.dqs.push(request)
-        except ValueError as e:  # non serializable request
+        except SerializationError as e:  # non serializable request
```

Comment:
In the topic of backward compatibility, should we consider to continue catching `ValueError` but log a deprecation warning when it is caught?